README.md

```diff
@@ -104,12 +109,67 @@ tree -L 1 ~/code/exercism
 
 #### Regenerating a Test Suite
 
-From within the ruby directory, run the following command:
+From time to time, the [canonical data](https://github.com/exercism/problem-specifications/tree/master/exercises)
+for an exercise's tests changes, and we need to keep the Ruby version's
+tests synced up.  Regenerating these tests is a quick and easy way to help
+maintain the track and get involved!
 
-    bin/generate --update <slug> 
+If it's your first time cloning/contributing to the repository, you'll
+need to install any dependencies via `bundle`:
 
-Leaving out the --update option will cause the BookKeeping version number to remain the same.
-This can be useful when testing generators.
+```bash
+bundle install
+```
+
+Be sure that you're working on the most up-to-date version of the repo.
+From the root of your copy of the repository:
+
+```bash
+# Add the exercism repo as upstream if you haven't yet:
+git remote add upstream https://github.com/exercism/ruby.git
+
```

Comment:
It could be mentioned that this only needs to be done once.

Reply:
Isn't this covered by: 
> if you haven't yet